cabby/geo/walk.py

```diff
@@ -141,25 +141,92 @@ def compute_route_from_points(self,
 
     return route_nodes
 
+  def get_non_specific_tag(self, poi):
```

Comment:
I think 'generic' is a better term than 'non_specific'.

Reply:
Changed